File: java&html/src/main/java/pt/unl/fct/di/apdc/fctconnect/util/Rooms/ReservationData.java
package pt.unl.fct.di.apdc.fctconnect.util.Rooms;

public class ReservationData {

    String user;
    String room;
    long hour;
    long day;

    public ReservationData() {}

    public ReservationData(String user, String room, long day, long hour) {
    	this.user = user;
    	this.room = room;
    	this.day = day;
        this.hour = hour;
    }

    public String getUser() {
    	return user;
    }

    public String getRoom() {
    	return room;
    }

    public long getDay() {
    	return day;
    }

    public long getHour() {
    	return hour;
    }


}

File: flutter/fctconnect/lib/models/ReservationData.dart
class ReservationData {
    final String user;
    final String room;
    final int hour;
    final int day;

    ReservationData({
        required this.user,
        required this.room,
        required this.hour,
        required this.day,
    });

    factory ReservationData.fromJson(Map<String, dynamic> json) {
        return ReservationData(
            user: json["user"],
            room: json["room"],
            hour: json["hour"],
            day: json["day"],
        );
    }
}